documentation/modules/proc-manual-restart-mirrormaker2-connector-task.adoc

```diff
@@ -14,19 +13,33 @@ This procedure describes how to manually trigger a restart of a Kafka MirrorMake
 .Procedure
 
 . Find the name of the `KafkaMirrorMaker2` custom resource that controls the Kafka MirrorMaker 2.0 connector you want to restart:
-[source,shell,subs=+quotes]
++
+[source,shell,subs="+quotes"]
+----
 kubectl get KafkaMirrorMaker2
-
-. Find the name of the Kafka MirrorMaker 2.0 connector and the ID of the task to be restarted from the `KafkaMirrorMaker2` custom resource.
-Task IDs are non-negative integers, starting from 0:
-[source,shell,subs=+quotes]
-kubectl describe KafkaMirrorMaker2 _KafkaMirrorMaker2-name_
-
-. To restart the connector task, annotate the `KafkaMirrorMaker2` resource in Kubernetes.
-For example, using `kubectl annotate` to restart task 0 of the connector named `my-source->my-target.MirrorSourceConnector`:
-[source,shell,subs=+quotes]
-kubectl annotate KafkaMirrorMaker2 _KafkaMirrorMaker2-name_ "strimzi.io/restart-connector-task=my-source->my-target.MirrorSourceConnector:0"
+----
+
+. Find the name of the Kafka MirrorMaker 2.0 connector and the ID of the task to be restarted from the `KafkaMirrorMaker2` custom resource. 
+Task IDs are non-negative integers, starting from 0.
++
+[source,shell,subs="+quotes"]
+----
+kubectl describe KafkaMirrorMaker2 _KAFKAMIRRORMAKER-2-NAME_
+----
+
+. To restart the connector task, annotate the `KafkaMirrorMaker2` resource in Kubernetes. 
+For example, using `kubectl annotate` to restart task 0 of the connector named `+my-source->my-target.MirrorSourceConnector+`:
```

Comment:
```suggestion
In this example, `kubectl annotate` is used to restart task _0_ of a connector named `+my-source->my-target.MirrorSourceConnector+`:
```

Reply:
The _->_ part of the name shows as arrow in the strimzi doc

Reply:
After some digging in the Asciidoctor user manual, I added the `+` symbols.

[Escape unconstrained formatting marks](https://docs.asciidoctor.org/asciidoc/latest/text/troubleshoot-unconstrained-formatting/#escape-unconstrained)

"Everything between the plus symbols is escaped from interpolation (attribute references, text formatting, etc.)"